python3-pyelftools: Depend on debugger, pprint

chaitu236 opened the pull request:
python3-pyelftools uses python3-debugger, python3-pprint.
So add dependencies on these packages.

@ni/rtos

amstewart commented:
PR looks good to me! :ok_hand: 

Since this change isn't needed for one of our releases, could you submit it to upstream first and then cherry-pick upstream's accepted version into ni/oe-core? That will minimize the burden of rebasing the patch.

chaitu236 commented:
Sure, I'll send this patch to upstream now

chaitu236 commented:
https://lists.openembedded.org/g/openembedded-core/message/159796.

chaitu236 committed:
python3-pyelftools: Depend on debugger, pprint

python3-pyelftools uses python3-debugger, python3-pprint.
So add dependencies on these packages.

Signed-off-by: Chaitanya Vadrevu <chaitanya.vadrevu@ni.com>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
(cherry picked from commit 22e84cdd05870f1a19c6389b66c4dfd5e9b418f7)
Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>

anujm1 committed:
python3-pyelftools: fix the override syntax

An earlier patch cherry-picked from master used : for override. Change
it to use _ for hardknott.

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>

chaitu236 commented:
@amstewart Changes are merged upstream. Cherry-picked [this](https://git.openembedded.org/openembedded-core/commit/?h=hardknott&id=66211faf4724b2c88eb4595e41fe98f5da96c3ee) and [this](https://git.openembedded.org/openembedded-core/commit/?h=hardknott&id=7e569186820163d731cbb14f8c25ce6a2cc45dc9).
